canton/BUILD.bazel

```diff
@@ -1024,6 +1102,7 @@ scala_library(
         "-language:postfixOps",
     ],
     unused_dependency_checker_mode = "error",
+    runtime_deps = [":community_reference-driver"],
```

Comment:
Is this expected with Canton v3? Might be worth writing a comment.

Reply:
Yes, this is the canton community reference driver for x nodes. I'm not sure a comment is warranted as all our public examples will use that one.